### PR TITLE
Added Threshold and Resource pages

### PR DESCRIPTION
**Pull request by jduncansma**
Added sub-folders for Library/Resources Library/Thresholds to UI folder and Images folder
Modified sidebars.js to include Library sub-menu under Solution Manager with links to Resources and Thresholds

### CODE REVIEW

File: docs/Files/UI/Solution-Manager/Library/Resources/Resources.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Resources
+
+Available Resources in OpCon are shown in the following Grid under Library -> Resources.
+![Resource Grid](../../../../../Resources/Images/SM/Library/Resources/Resource-Grid.png "Resource Grid")
+
+Clicking the **Add** or **Edit** button bring up the following screen:
+![Resource Add](../../../../../Resources/Images/SM/Library/Resources/Resource-Add.png "Resource Add Screen")
+
+![White pencil/paper icon on gray circular background](<../../../../../Resources/Images/note-icon(48x48).png> "Note icon") **NOTE:** [The **Name** field is unique across Resources _and_ Thresholds
```
Comment (ryandimick) [inline docs/Files/UI/Solution-Manager/Library/Resources/Resources.md]: Can we please use a notation like this for notes on a go-forward basis?

All notes in the format provided here will be phased out.

:::note
your text here
:::

File: docs/Files/UI/Solution-Manager/Library/Thresholds/Thresholds.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Thresholds
+
+Available Thresholds in OpCon are shown in the following Grid under Library -> Thresholds.
+![Resource Grid](../../../../../Resources/Images/SM/Library/Thresholds/Threshold-Grid.png "Threshold Grid")
+
+Clicking the **Add** or **Edit** button bring up the following screen:
+![Resource Add](../../../../../Resources/Images/SM/Library/Thresholds/Threshold-Add.png "Threshold Add Screen")
+
+![White pencil/paper icon on gray circular background](<../../../../../Resources/Images/note-icon(48x48).png> "Note icon") **NOTE:** [The **Name** field is unique across Resources _and_ Thresholds
```
Comment (ryandimick) [inline docs/Files/UI/Solution-Manager/Library/Thresholds/Thresholds.md]: Same as above, use:

:::note
text
:::

**Review by ryandimick — CHANGES_REQUESTED** [review]

File: docs/Files/UI/Solution-Manager/Library/Resources/Resources.md (hunk 1 above)
Comment (jduncansma) [reply docs/Files/UI/Solution-Manager/Library/Resources/Resources.md]: Do we have any guides / best practices for this in confluence or anywhere? I think I copied this from another page but it must have been an older version.

Comment (ryandimick) [reply docs/Files/UI/Solution-Manager/Library/Resources/Resources.md]: Contributor guide is hosted here, the style guide needs to be built out though: https://help.smatechnologies.com/opcon/contributor-guide/

**Review by ryandimick — APPROVED** [review]

**Pull request merged**